zachsinterview/src/main/java/com/imprev/zachsinterview/Main.java

```diff
@@ -0,0 +1,37 @@
+package com.imprev.zachsinterview;
+
+public class Main {
+
+    /**
+     * Demonstrates how to connect to thirdPartyAPI and tests how the numerical key store may be used.
+     */
+    public static void main(String[] argv) throws NumberStoreException {
+        
+        String username = "moxiworks";
+        String password = "fjdskal3kd!jdkdInterviewFun!!!";
+        String apiUrl = "https://example.com/thirdPartyApiURL";
+        
+        NumberStore numberStore = new NumberStoreImpl(username, password, apiUrl , 80);
```

Comment:
all parameters aside from (presumably) port number are passed as variables instead of hard coded values, would be good to do that for port number too for consistency.